wolfssl/certs_test_sm.h

```diff
@@ -0,0 +1,8400 @@
+/* certs_test_sm.h */
+/* This file was generated using: ./gencertbuf_sm.pl */
```

Comment:
was gencertbuf_sm.pl supposed to be in this PR?

Reply:
Yes, see https://github.com/wolfSSL/wolfssl/issues/6681 although the correct name is [gencert_sm.sh](https://github.com/gojimmypi/wolfssl/blob/ED25519_SHA2_fix/gencert_sm.sh) and needs to be cleaned up.